packages/settings-library/src/settings_library/base.py

```diff
@@ -30,12 +33,26 @@ def _set_defaults_with_default_constructors(
 
         # Builds defaults at this point
         for name, default_cls in default_fields:
-            with suppress(ValidationError):
+            try:
                 default = default_cls.create_from_envs()
                 field_obj = cls.__fields__[name]
                 field_obj.default = default
                 field_obj.field_info.default = default
                 field_obj.required = False
+            except ValidationError as e:
```

Comment:
Will check with him because I also found it strange. Will ask @pcrespov to provide a test for where the suppress case was useful. Otherwise it stays as it is.